### PR TITLE
C357064 Verify Items bulk edit if the location name contains "/" (firebird) (TaaS)

### PR DESCRIPTION
**Pull request by KarapetyanSofia**
[C357064](https://foliotest.testrail.io/index.php?/cases/view/357064)

[Allure Report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5561/allure)

Screenshot 2023-11-21 at 4 36 48 PM

### CODE REVIEW

File: cypress/e2e/bulk-edit/csv/bulk-edit-location.cy.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        item.barcode,
+        'Permanent Location',
+        'Temporary Location',
+      ]);
+      // Click "Actions" menu => Select "Start Bulk edit" option
+      BulkEditActions.openInAppStartBulkEditFrom();
+      // Select "Temporary item location" from "Select option" dropdown => Select "Replace with" option from  "Select action" dropdown
+      BulkEditActions.replaceTemporaryLocation(locationName);
+      // Click on the "Plus" icon
+      BulkEditActions.addNewBulkEditFilterString();
```
Comment (manowarqqq) [inline cypress/e2e/bulk-edit/csv/bulk-edit-location.cy.js:101]: no expected results from Step 9

File: cypress/e2e/bulk-edit/csv/bulk-edit-location.cy.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+      // Select "Permanent item location" from  "Select option" dropdown on the new row => Select "Replace with" option from  "Select action" dropdown
+      BulkEditActions.replacePermanentLocation(locationName, 'item', 1);
+      // Click "Confirm changes" button
+      BulkEditActions.confirmChanges();
```
Comment (manowarqqq) [inline cypress/e2e/bulk-edit/csv/bulk-edit-location.cy.js:105]: Review all step's expected results. Lack of assertions

**Review by manowarqqq — CHANGES_REQUESTED** [review]

File: cypress/e2e/bulk-edit/csv/bulk-edit-location.cy.js (hunk 1 above)
Comment (ernestyedigaryan1) [reply cypress/e2e/bulk-edit/csv/bulk-edit-location.cy.js:101]: Done!

File: cypress/e2e/bulk-edit/csv/bulk-edit-location.cy.js (hunk 2 above)
Comment (ernestyedigaryan1) [reply cypress/e2e/bulk-edit/csv/bulk-edit-location.cy.js:106]: Done!

**Review by manowarqqq — APPROVED** [review]

File: cypress/e2e/bulk-edit/csv/bulk-edit-location.cy.js
Comment (nayimovag) [inline cypress/e2e/bulk-edit/csv/bulk-edit-location.cy.js:1]: move this to bulk-edit/in-app folder please, and make the file name more specific

File: cypress/e2e/bulk-edit/csv/bulk-edit-location.cy.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+import ItemRecordView from '../../../support/fragments/inventory/item/itemRecordView';
+import ServicePoints from '../../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import FileManager from '../../../support/utils/fileManager';
+import ExportFile from '../../../support/fragments/data-export/exportFile';
+import DateTools from '../../../support/utils/dateTools';
+import Location from '../../../support/fragments/settings/tenant/locations/newLocation';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+
+describe('bulk-edit', () => {
```
Comment (nayimovag) [inline cypress/e2e/bulk-edit/csv/bulk-edit-location.cy.js:17]: we need 2 describe blocks here, like this
describe('bulk-edit', () => {
  describe('in-app approach', () => {

File: cypress/e2e/bulk-edit/csv/bulk-edit-location.cy.js [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+  const testData = {
+    folioInstances: InventoryInstances.generateFolioInstances(),
```
Comment (nayimovag) [inline cypress/e2e/bulk-edit/csv/bulk-edit-location.cy.js:19]: are you using this?

**Review by nayimovag — APPROVED** [review]

**Pull request merged**